spec/fortnox/api/repositories/invoice_spec.rb

```diff
@@ -1,10 +1,20 @@
 require 'spec_helper'
-require 'fortnox/api/repositories/context'
+require 'fortnox/api/repositories/environment_context'
 require 'fortnox/api/repositories/invoice'
-require 'fortnox/api/models/invoice'
+require 'fortnox/api/repositories/examples/all_examples'
+require 'fortnox/api/repositories/examples/find_examples'
+require 'fortnox/api/repositories/examples/save_examples'
 
 describe Fortnox::API::Repository::Invoice do
-  include_context 'repository context'
+  include_context 'environment context'
 
-  it_behaves_like 'repositories', Fortnox::API::Model::Invoice
+  include_examples '.all'
+
+  include_examples '.find'
+
+  include_examples '.save', :comments do
+    let( :value ){ 'An invoice comment' }
+    let( :updated_value ){ 'An updated invoice comment' }
+    let( :model_hash ){ { comments: value, customer_number: 1 } }
```

Comment:
If we can assume that all models have a text field that can include whatever text we provide, then I guess we can safely remove `:value` and `:updated_value`.

I think it is harder to remove `model_hash`. That hash will look differently depending on what attributes are required for the current model. For instance, `Customer` requires `:name` and `Invoice` requires `customer_number`. But if you have any solution to this, please let me know and I'll implement it :)